docs/UserGuide.md

```diff
@@ -28,7 +31,7 @@ This document is a comprehensive guide to all the commands available to you, alo
 
 Step 1. Ensure you have Java `11` or above installed in your computer. You can download it from [here](https://www.oracle.com/sg/java/technologies/javase/jdk11-archive-downloads.html).
```

Comment:
I think maybe we can change "from here" to "from this website", since we changed the phrasing of the line below

Reply:
I changed it to "from the [Oracle Website]", if that's ok?